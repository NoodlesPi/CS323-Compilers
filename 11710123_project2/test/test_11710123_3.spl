// type 3
int main(){
    int a = 2;
    int a = 3;
}