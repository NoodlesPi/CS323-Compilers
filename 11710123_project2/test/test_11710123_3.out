Error type 3 at Line 4: variable is redefined in the same scope
